ramls/schemas/jobExecution.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema":"http://json-schema.org/draft-04/schema#",
+  "description":"Job Execution Schema",
+  "type":"object",
+  "properties":{
+    "id":{
+      "description":"Unique identifier",
+      "$ref":"../../ramls/raml-util/schemas/uuid.schema"
+    },
+    "hrId":{
+      "description":"Human readable id",
+      "type":"string"
+    },
+    "fileName":{
+      "description":"File name",
+      "type":"string"
+    },
+    "jobProfile":{
+      "description":"Related JobProfile information",
+      "type":"object",
+      "$ref":"jobProfile.json"
+    },
+    "recordsCount":{
+      "description":"Number of exported records",
+      "type":"integer"
+    },
+    "endedRunning":{
+      "description":"Date and time when the job execution completed",
+      "type":"string",
+      "format":"date-time"
+    },
+    "runBy":{
+      "description":"First and last name of the user who triggered the job execution",
+      "type":"object",
+      "properties":{
+        "firstName":{
+          "description":"First name",
+          "type":"string"
+        },
+        "lastName":{
+          "description":"Last name",
+          "type":"string"
+        }
+      }
+    },
+    "status":{
+      "description":"Status of files upload",
```

Comment:
Should this be status of upload or export?

Reply:
Please fix description, this should be about export